isis/tests/Fixtures.cpp

```diff
@@ -202,4 +201,96 @@ namespace Isis {
     delete threeImageOverlapFile; 
     delete twoImageOverlapFile; 
   }
+
```

Comment:
Should we find a way to better organize these one-off fixtures? This file is going to get crowded if we keep adding 60+ engest programs to it, plus any others that come along.

Reply:
I was thinking there is a way to make the mission specific camera stuff more generic, but after looking at options it seems we will inevitably end up with a lot of fixtures for different missions. @kaitlyndlee and I briefly talked about this, I personally don't mind letting Fixtures.h get long, but if we wanted a `MissionFixtures.h/.cpp` I wouldn't be against it.